### PR TITLE
PL update rule: add new vaccine - Nuvaxovid

### PR DESCRIPTION
**Pull request by rafalgra**

### CODE REVIEW

File: PL/VR-PL-0000/rule.json [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
+      "desc": "Akceptowane są tylko szczepionki z zestawu wartości, które zostały zatwierdzone przez EMA: Comirnaty, Moderna (Spikevax), Vaxzevria (AstraZeneca), Jannsen, Nuvaxovid."
     },    
 	{
       "lang": "en",
       "desc": "Only vaccines in the allowed valueset that have been approved by the EMA are allowed."
     },
     {
       "lang": "de",
-      "desc": "Nur die folgenden Impfstoffe werden akzeptiert: AstraZeneca, Biontech, Janssen, Moderna."
+      "desc": "Nur die folgenden Impfstoffe werden akzeptiert: Vaxzevria (AstraZeneca), Comirnaty, Janssen, Moderna (Spikevax), Nuvaxovid."
```
Comment (dslmeinte) [inline PL/VR-PL-0000/rule.json:21]: You could consider putting "(Pfizer)" behind "Comirnaty" for consistency.

File: PL/VR-PL-0000/rule.json [hunk 2]
```diff
@@ -41,7 +41,8 @@
             "EU/1/20/1528",
             "EU/1/20/1507",
             "EU/1/21/1529",
-            "EU/1/20/1525"
+            "EU/1/20/1525",
+			"EU/1/21/1618"
```
Comment (dslmeinte) [inline PL/VR-PL-0000/rule.json:45]: Did you consider also allowing the old coding for Nuvaxovid (NVX-CoV2373)? The vaccine is the same, it's just the coding which didn't exist before.

File: PL/VR-PL-0000/rule.json (hunk 1 above)
Comment (rafalgra) [reply PL/VR-PL-0000/rule.json]: done

File: PL/VR-PL-0000/rule.json (hunk 2 above)
Comment (rafalgra) [reply PL/VR-PL-0000/rule.json:45]: We considered it, but decided to add new code only (at least for now)

**Comment by dslmeinte** [comment]
@rafalgra Can I merge for you?

**Comment by rafalgra** [comment]
Yes

**Comment by rafalgra** [comment]
> 

yes

**Pull request merged**